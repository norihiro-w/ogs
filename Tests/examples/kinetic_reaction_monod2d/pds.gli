#POINTS
0 0 0 0 
1 5.00 0 0 
2 5.00 0.2 0 
3 0 0.2 0 
4 0 0.125 0 
5 0 0.075 0 
6 1.0 0.0 0 
7 1.0 0.2 0 
8 1.0 0.1 0 
9 0.0 0.13 0 
10 0.0 0.07 0 
11 5.00 0.15 0 
12 5.00 0.155 0 
#POLYLINE
 $NAME
  PLY_BC_RIGHT
 $EPSILON
  0.0001
 $POINTS
  1
  2
#POLYLINE
 $NAME
  PLY_BC_LEFT
 $EPSILON
  0.0001
 $POINTS
  0
  3
#POLYLINE
 $NAME
  PLY_S1
 $EPSILON
  0.0001
 $POINTS
  3
  9
#POLYLINE
 $NAME
  PLY_S2
 $EPSILON
  0.0001
 $POINTS
  4
  5
#POLYLINE
 $NAME
  PLY_S3
 $EPSILON
  0.0001
 $POINTS
  0
  10
#POLYLINE
 $NAME
  PLY_4
 $EPSILON
  0.0001
 $POINTS
  6
  7
#POLYLINE
 $NAME
  PLY_OUT1
 $EPSILON
  0.0001
 $POINTS
  1
  11
#POLYLINE
 $NAME
  PLY_OUT2
 $EPSILON
  0.0001
 $POINTS
  12
  2
#STOP
