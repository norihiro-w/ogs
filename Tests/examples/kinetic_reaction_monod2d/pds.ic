GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1.0
  
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_A
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0
  
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_B
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 2.5e-4 
  
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_C
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0
  
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Biomass
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1.0e-6

#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  ConsTracer
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0
#STOP

#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Cons_Tracer
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0