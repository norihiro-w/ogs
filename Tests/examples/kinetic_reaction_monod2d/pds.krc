GeoSys Kinetic Reactions
#KINREACTIONDATA      
 $SOLVER_TYPE
	1                ; Solver type
 $RELATIVE_ERROR	
	1.e-6		 ; rel. Error
 $MIN_TIMESTEP	
	1.e-6		; min Timestep
 $INITIAL_TIMESTEP	
	1.e-4       ; init Timestep
 $BACTERIACAPACITY
  100.0
 $NO_REACTIONS
   POLYLINE PLY_BC_LEFT

#REACTION ; 
 $NAME
   DoubleMonod ;
 $TYPE
   monod;
 $BACTERIANAME
   Biomass
 $EQUATION
  1 Comp_A + 1 Comp_B =  1 Comp_C; 
 $RATECONSTANT
 1.15741e-5  1.0   ; max growth rate my_max , exponent
 $GROWTH
    1
 $MONODTERMS
  ;2   ; Number of monod terms in kinetic equation
  Comp_A   8.33e-5  1.0 ; for each monod term: species concentration order
  Comp_B   3.13e-5  1.0
 $INHIBITIONTERMS
  ;0
 $PRODUCTIONSTOCH
  ;2
  Comp_A  -1.0
  Comp_B  -1.0
  Comp_C  +1.0
  
#REACTION ; 
 $NAME
   DoubleMonodDecay ;
 $TYPE
   monod;
 $BACTERIANAME
   Biomass
 $EQUATION
  1 Biomass =  1 Biomass; 
 $RATECONSTANT
 1.15741e-5  1.0   ; max growth rate my_max , exponent
 $GROWTH
    1
 $MONODTERMS
  ;2   ; Number of monod terms in kinetic equation
  Comp_A   8.33e-5  1.0 ; for each monod term: species concentration order
  Comp_B   3.13e-5  1.0
 $DECAYTERM
  1.15741e-6  ; 
 $INHIBITIONTERMS
  ;0
 $PRODUCTIONTERMS
  ;0
  
#STOP


