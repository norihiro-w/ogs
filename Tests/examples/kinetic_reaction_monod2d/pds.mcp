#COMPONENT_PROPERTIES  ; comp0
 $NAME  ; 
  ConsTracer ; 
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0            	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp1
 $NAME
  Comp_A ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp2
 $NAME
  Comp_B ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp3
  $NAME
 Comp_C ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0            	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp4
  $NAME
  Biomass ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  0             	; diffusion model type, diffusion constant
 
#STOP

; temperary disable the tracer for now
