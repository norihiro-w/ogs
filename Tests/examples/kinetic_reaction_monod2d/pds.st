GeoSys-ST: Source Terms ------------------------------------------------
#SOURCE_TERM
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE PLY_BC_RIGHT
 $DIS_TYPE
  CONSTANT_NEUMANN  -1.157407407e-5 ; q=1.0m/d
#STOP


