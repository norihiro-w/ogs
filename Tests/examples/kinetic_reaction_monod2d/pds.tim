GeoSys-TIM: Timestepping ------------------------------------------------
#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   1    120.0
   1    8639880.0
 $TIME_END
  8640000.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  REACT_GIA
 $TIME_END
  8640000.0
 $TIME_START
  0.0
$TIME_CONTROL
   NEWTON_ADAPTIVE
 MAX_TIME_STEP
     8640
 MIN_TIME_STEP
     120.0
 ITER_TIMES_AND_MULTIPLYER
   4
   0    1.6
   1    1.0
   3    0.5
   7   0.25  
#STOP






#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   1    120.0
   1    8639880.0
 $TIME_END
  8640000.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  REACT_GIA
 $TIME_END
  8640000.0
 $TIME_START
  0.0
$TIME_CONTROL
   NEWTON_ADAPTIVE
 MAX_TIME_STEP
     86400
 MIN_TIME_STEP
     120.0
 ITER_TIMES_AND_MULTIPLYER
   4
   0    1.6
   3    1.0
   7   0.5
   12   0.25  


#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   100000  864.0
 $TIME_END
  8640000.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
    REACT_GIA
 $TIME_STEPS
   100000  864.0
 $TIME_END
  8640000.0
 $TIME_START
  0.0
