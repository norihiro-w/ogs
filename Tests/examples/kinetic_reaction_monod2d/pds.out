GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  ConsTracer
  Comp_A
  Comp_B
  Comp_C
  Biomass
  eta_mob_0
  eta_mob_1
  eta_mob_2
  xi_mob_0
  xi_immob_0
  xi_mob_rate_0
  xi_immob_rate_0
 $ELE_VALUES
  VELOCITY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 10

#OUTPUT // profile
 $NOD_VALUES
  HEAD
  ConsTracer
  Comp_A
  Comp_B
  Comp_C
  Biomass
  eta_mob_0
  eta_mob_1
  eta_mob_2
  xi_mob_0
  xi_immob_0
  xi_mob_rate_0
  xi_immob_rate_0
 $ELE_VALUES
  VELOCITY
 $GEO_TYPE
  POLYLINE PLY_4
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 10
#STOP

